bin/run.sh

```diff
@@ -74,7 +74,7 @@ run_tests() {
 
     cd "$solution_dir"
 
-    sed -i 's/load bats-extra.bash/load bats-extra/' "$test_file"
+    sed -i 's/load bats-extra.bash/load bats-extra/' "$test_file" || true
```

Comment:
Does `sed` fail when there's no changes? Or when it can't find the file? Should this be testing for file existance?

Reply:
@IsaacG : `sed` exits wist status 2 when input file is missing (`set -o errexit`). It does not when there is no change. The expected error string is `bats`'s one on missing test file (this is why we need to continue here).